assets/amazon.yaml

```diff
@@ -86,3 +86,42 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: https://github.com/amazon-science/chronos-forecasting/discussions
+- type: model
+  name: Amazon Nova
+  organization: Amazon Web Services (AWS)
+  description: A new generation of state-of-the-art foundation models (FMs) that
+    deliver frontier intelligence and industry leading price performance, available
+    exclusively in Amazon Bedrock. You can use Amazon Nova to lower costs and latency
+    for almost any generative AI task.
+  created_date: 2024-12-03
+  url: https://aws.amazon.com/blogs/aws/introducing-amazon-nova-frontier-intelligence-and-industry-leading-price-performance/
+  model_card: unknown
+  modality:
+    explanation: Amazon Nova understanding models accept text, image, or video inputs
```

Comment:
Maybe split into 2 model cards because they differ in modalities

i.e. Nova understanding vs Nova creative content generation